sky130-build/build.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse
+import copy
+import siliconcompiler
+import os
+
+from sources import add_sources
+from libs import lib_setup
+
+def setup_options(chip):
+        chip.set('option', 'define', 'VERILATOR')
+        chip.set('option', 'loglevel', 'INFO')
+        chip.set('option', 'relax', True)
+        chip.set('option', 'jobname', 'job0')
+
+        chip.add('option', 'define', 'WT_DCACHE')
+        
+        cur_dir = os.path.dirname(os.path.realpath(__file__))
+        chip.add('option', 'define', f'MEM_ROOT={cur_dir}')
```

Comment:
I think you can delete these lines, they were in ZeroSoC to provide a memory file for simulation/testing on an FPGA.